src/Routing/Filter/ControllerEndpointFilter.php

```diff
@@ -9,6 +9,7 @@
  * This filter registers the endpoint model type in controllers
  *
  * @package Muffin\Webservice\Routing\Filter
+ * @deprecated Since 2.0.0
```

Comment:
The format for deprecated tag is `@deprecated <version> <description>`

Reply:
Whoops. That's me typing too fast, I've done that in #63 also.